golang-job-type/go_wrapper/go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/TheRacetrack/plugin-go-job-type/golang-job-type
 
-go 1.16
+go 1.20
 
 require (
-	github.com/gorilla/mux v1.8.0
+    github.com/gin-gonic/gin v1.9.0
```

Comment:
Since you updated `go.mod`, you should also push your changes to `go.sum`. Now it produces ` missing go.sum entry` for me